config/jobs/etcd/etcd-presubmits.yaml

```diff
@@ -707,3 +707,65 @@ presubmits:
           limits:
             cpu: "6"
             memory: "4Gi"
+
+  - name: pull-etcd-grpcproxy-integration
+    optional: true # remove once the job is stable
+    cluster: eks-prow-build-cluster
+    always_run: false # set to true once the job works
+    branches:
+      - main
+      - release-3.6
+    decorate: true
+    decoration_config:
+      timeout: 60m
+    annotations:
+      testgrid-dashboards: sig-etcd-presubmits
+      testgrid-tab-name: pull-etcd-grpcproxy-integration
+    spec:
+      containers:
+      - image: us-central1-docker.pkg.dev/k8s-staging-test-infra/images/kubekins-e2e:v20250205-f1f3519e6b-master
+        command:
+          - runner.sh
+        args:
+          - bash
+          - -c
+          - |
+            CPU=4 make test-grpcproxy-integration
+        resources:
+          requests:
+            cpu: "6"
+            memory: "4Gi"
+          limits:
+            cpu: "6"
+            memory: "4Gi"
+
+  - name: pull-etcd-grpcproxy-e2e
+    optional: true # remove once the job is stable
+    cluster: eks-prow-build-cluster
```

Comment:
Should we be running these tests on `arm64` as well for parity?

Reply:
You mean parity with the prow jobs, right? Because we only had them running for AMD64 in the GitHub workflows.

I'll make the change, it makes sense.

Reply:
Would you suggest having `pull-etcd-contrib-mixin` or `pull-etcd-fuzzing-v3rpc`, multi-arch too?